Add laravel-s library (#1082)
ziadoz/awesome-php: README.md

* [CakePHP CRUD](https://github.com/friendsofcake/crud) - A Rapid Application Development (RAD) plugin for CakePHP.
* [Knp RAD Components](https://rad.knplabs.com/) - A set of Rapid Application Development (RAD) components for Symfony.

> * [LaravelS](https://github.com/hhxsv5/laravel-s) - A glue that is used to quickly integrate Swoole into Laravel or Lumen, and then give them better performance and more possibilities.

Reviewer: ```suggestion
* [LaravelS](https://github.com/hhxsv5/laravel-s) - Glue for using Swoole in Laravel or Lumen.
```
